rpcserver.go

```diff
@@ -408,7 +408,9 @@ func (r *rpcServer) sendCoinsOnChain(paymentMap map[string]int64,
 		return nil, err
 	}
 
-	return r.server.cc.wallet.SendOutputs(outputs, feeRate)
+	tx, err := r.server.cc.wallet.SendOutputs(outputs, feeRate)
+	txHash := tx.TxHash()
```

Comment:
I believe this will panic if the error is non-nil due to a nil `wire.MsgTx` being returned, so we want to check the error before doing this.

Reply:
Fixed